addons/mail/migrations/13.0.1.0/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,115 @@
+# Nothing to do
+
+mail         / mail.message             / message_type (selection)      : selection_keys is now '['comment', 'email', 'notification', 'user_notification']' ('['comment', 'email', 'notification']')
+# I think nothing to do
+# PR https://github.com/odoo/odoo/pull/32404/commits
+# Commit https://github.com/odoo/odoo/commit/d917b2d072a837d0d7deb0429b7552d680f26b6f
+
+mail         / mail.message             / email_layout_xmlid (char)     : NEW [renamed from layout]
+mail         / mail.message             / layout (char)                 : DEL [renamed to email_layout_xmlid]
+mail         / mail.message             / needaction_partner_ids (many2many): DEL relation: res.partner [renamed to notified_partner_ids]
+mail         / mail.message             / notified_partner_ids (many2many): NEW relation: res.partner [renamed from needaction_partner_ids]
+# Done in pre migration script
+
+mail         / mail.message             / res_id (integer)              : relation is now 'model' ('False') [nothing to do]
+mail         / mail.message             / res_id (integer)              : type is now 'many2one_reference' ('integer')
+# Nothing to do
+
+mail         / mail.message             / canned_response_ids (one2many): NEW relation: mail.shortcode [inverse for message_ids]
+mail         / mail.shortcode           / message_ids (many2one)        : NEW relation: mail.message
+# I think nothing to do
+# Commit https://github.com/odoo/odoo/commit/66a08369741369946df219ac92b8881711fbb484
+
+mail         / mail.notification        / email_status (selection)      : DEL selection_keys: ['bounce', 'canceled', 'exception', 'ready', 'sent'] [renamed to email_status]
+mail         / mail.notification        / notification_status (selection): NEW selection_keys: ['bounce', 'canceled', 'exception', 'ready', 'sent'], hasdefault [renamed from email_status]
+# Done in pre-migration script
+
+mail         / mail.notification        / notification_type (selection) : NEW required, selection_keys: ['email', 'inbox'], req_default: function, hasdefault
+# Done in post-migration script. Tips from commit: [notification_type: notification.is_email and 'email' else 'inbox']
+
+mail         / mail.notification        / read_date (datetime)          : NEW
+# I think nothing to do
+# Commit https://github.com/odoo/odoo/commit/0c37b9fb39005eab70ee8b39da84f79c981b2695
+
+mail         / mail.notification        / is_email (boolean)            : DEL
+# Nothing to do
+
+mail         / mail.tracking.value      / track_sequence (integer)      : DEL [renamed to tracking_sequence]
+mail         / mail.tracking.value      / tracking_sequence (integer)   : NEW hasdefault [renamed from track_sequence]
+# Done in pre-migration script
+
+mail         / res.partner              / email_normalized (char)       : NEW isfunction: function, stored
+# Check that Odoo core support this change
```

Comment:
Make that `Checked` or it sounds like a TODO

Reply:
It`s to do. I want to try local migration today.

Reply:
Ah I see! Odoo will for sure apply the compute method on new stored computed fields for all records.